numba/cuda/api.py

```diff
@@ -85,6 +85,14 @@ def is_cuda_array(obj):
     return hasattr(obj, '__cuda_array_interface__')
 
 
+def is_fp16_supported():
+    """ Determine if IEEE floating-point 16-bit is supported.
+
+        Returns True if fp16 is supported, False otherwise.
```

Comment:
As per comment above:

```suggestion
def is_float16_supported():
    """Whether 16-bit floats are supported.

    Returns True if float16 is supported, False otherwise.
```